components/definitions/rewards.d.ts

```diff
@@ -76,7 +76,7 @@ declare namespace Rewards {
     expiryTime: number
     captcha?: string
     hint?: string
-    status?: 'wrongPosition' | 'grantGone' | 'generalError' | number | null
+    status?: 'wrongPosition' | 'grantGone' | 'generalError' | 'grantAlreadyClaimed' | number | null
```

Comment:
This is now getting pretty long, let's put this one in it's own type, `GrantStatus`, wdyt?

Reply:
done